src/julia.h

```diff
@@ -286,6 +286,7 @@ typedef struct _jl_code_info_t {
     uint8_t inferred;
     uint8_t propagate_inbounds;
     uint8_t has_fcall;
+    uint8_t noinfer;
```

Comment:
So is the intent here that we might eventually be able to have both inferred and noinferred compilations of the same method and thus support call-site `@noinfer` annotations? If so, 👍

Reply:
Currently we represent boolean properties as `uint8_t` field, including `noinfer`, but yes, it's possible to implement call-site `@noinfer`.